modules/microshift-audit-logs-config-proc.adoc

```diff
@@ -0,0 +1,66 @@
+// Text snippet included in the following assemblies:
+//
+// * microshift_configuring/microshift-audit-logs-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-audit-log-values_{context}"]
+= Configuring audit log values
+
+Use the {microshift-short} service configuration file to implement custom audit log settings.
+
+.Procedure
+
+. Add the following `apiServer` stanza to your {microshift-short} `config.yaml` file. If you need to make the file, run the following command:
++
+[source,terminal]
+----
+$ sudo cat /etc/microshift/config.yaml
+----
++
+.Example `auditLog` configuration
+[source,yaml]
+----
+apiServer:
+# ....
+  auditLog:
+    maxFileSize: 200 # <1>
+    maxFiles: 1 # <2>
+    maxFileAge: 7 # <3>
+    profile: Default # <4>
+# ....
+----
+<1> The maximum audit log file size in megabytes. If the value is 0, the limit is disabled. In this example, after the live log reaches the 200Mb limit, it is rotated.
+<2> The maximum number of rotated audit log files to retain. After the limit is reached, the log files in order from oldest to newest are deleted until the specified limits are reached. When the value is 0, the limit is disabled. In this example, the value `1` results in only 1 file of size `maxFileSize` being retained in addition to the current active log.
+<3> Specifies the maximum time in days that log files are kept. Files older than this limit are deleted. When the value is 0, the limit is disabled. In this example, after a log file is more than 7 days old, it is deleted. The deletion happens regardless of whether the live log has reached the maximum file size as give in the `maxFileSize` field value. File age is determined by the timestamp written in the name of the rotated log file, for example, `audit-2024-05-16T17-03-59.994.log`.
+<4> Logs only metadata for read and write requests; does not log request bodies except for OAuth access token requests. If you do not specify this field, the Default profile is used.
+
+. Optional: To specify a specific directory for logs, you can stop {microshift-short}, then move the `/var/log/kube-apiserver` directory to your desired location:
+
+.. Stop {microshift-short} by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl stop microshift
+----
+.. Move the `/var/log/kube-apiserver` directory to your desired location by running the following command:
++
+[source,terminal]
+----
+$ sudo mv /var/log/kube-apiserver <~/kube-apiserver> <1>
+----
+<1> Replace _<~/kube-apiserver>_ with the path to the directory that you want to use.
+
+. If you specified a specific directory for logs, create a symlink to your custom directory at `/var/log/kube-apiserver` by running the following command:
```

Comment:
Optional ??

Reply:
Not _really_. We want logs to be accessible to sos reports and this is how we achieve that.